### PR TITLE
Added refactoring information of doc

### PR DESCRIPTION
**Pull request by s-yadav**
Hey @ilearnio,
Updated readme to add information for refactoring code to start using alias.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,3 +142,7 @@ In order to register a custom modules path (`addPath`) it modifies the internal
 [npm-url]: https://npmjs.org/package/module-alias
 [travis-image]: https://img.shields.io/travis/ilearnio/module-alias/master.svg
 [travis-url]: https://travis-ci.org/ilearnio/module-alias
+
+## Refactor your code
```
Comment (ilearnio) [inline README.md]:
```suggestion
## Refactor your code (for already existing projects)
```

File: README.md [hunk 2]
```diff
@@ -142,3 +142,7 @@ In order to register a custom modules path (`addPath`) it modifies the internal
...
+
+If you are using this on an existing project, you can use [relative-to-alias](https://github.com/s-yadav/relative-to-alias) to refactor your code to start using the alias.
```
Comment (ilearnio) [inline README.md]:
```suggestion
If you are using this on an existing project, you can use [relative-to-alias](https://github.com/s-yadav/relative-to-alias) to refactor your code to start using aliases.

**Comment by ilearnio** [comment]
Hey, added a few suggestions

**Comment by s-yadav** [comment]
Hey @ilearnio . Thanks for reviewing the code. Made the requested changes.

**Comment by ilearnio** [comment]
Merged, I hope your package works good 😉 Didn't test it myself. But it looks like a good fit to the readme

**Comment by s-yadav** [comment]
Thanks, I have used it in few projects. Works well there.

**Pull request merged**